scss/layout.scss

```diff
@@ -0,0 +1,10 @@
+.layout {
+  display: flex;
+  flex-flow: row wrap;
+  margin: 30px 70px 100px 70px;
```

Comment:
the 30px should be a margin bottom added to the title, the side margins should be paddings since their objective is to separate each tech from its container. the 100px at the bottom is ok :3